### PR TITLE
Added support on pdf preview and radiology error on feeding input for paid status and insurance also gepg ammendment

### PR DESCRIPTION
**Pull request by iamgaspardev**

### CODE REVIEW

**Comment by Moricembiaji** [comment]
Thanks @Gaspar Giddson ***@***.***>

File: docker/build-local.sh
Comment (josephatJ) [inline docker/build-local.sh:1]: Seems you have changed the file to build local. This means it will fail to build

File: package-lock.json
Comment (josephatJ) [inline package-lock.json:1]: If this file is not within UI folder consider deleting it

File: package.json
Comment (josephatJ) [inline package.json:1]: This is supposed to be deleted

File: ui/src/app/modules/billing/components/bill-confirmation/bill-confirmation.component.ts
Comment (josephatJ) [inline ui/src/app/modules/billing/components/bill-confirmation/bill-confirmation.component.ts:1]: Please make sure you remove logs before going live

File: ui/src/app/shared/dialogs/shared-pdf-preview/shared-pdf-preview.component.ts
Comment (josephatJ) [inline ui/src/app/shared/dialogs/shared-pdf-preview/shared-pdf-preview.component.ts:1]: Remove console logs before pushing

**Review by josephatJ — CHANGES_REQUESTED** [review]
Please address the comments first before i approve

**Comment by Moricembiaji** [comment]
@Gaspar Giddson ***@***.***> Please work on it immediately
so that I can test it today.

**Review by josephatJ — COMMENTED** [review]
Noted

**Pull request merged**